plat/socionext/synquacer/sq_psci.c

```diff
@@ -33,12 +35,16 @@ uintptr_t sq_sec_entrypoint;
 
 int sq_pwr_domain_on(u_register_t mpidr)
 {
+#if SQ_USE_SCMI_DRIVER
+	css_scp_on(mpidr);
```

Comment:
As stated in my previous comment, instead of calling this function, implement a synq_scmi_on() function. This function can directly call the required SCMI driver.